pkg/controller/bookkeepercluster/bookkeepercluster_controller.go

```diff
@@ -234,31 +235,47 @@ func (r *ReconcileBookkeeperCluster) syncBookieSize(bk *bookkeeperv1alpha1.Bookk
 func (r *ReconcileBookkeeperCluster) reconcileFinalizers(bk *bookkeeperv1alpha1.BookkeeperCluster) (err error) {
 	if bk.DeletionTimestamp.IsZero() {
 		if !util.ContainsString(bk.ObjectMeta.Finalizers, util.ZkFinalizer) {
+			// appending name of pravega cluster to the finalizers
+			configMap := &corev1.ConfigMap{}
+			err = r.client.Get(context.TODO(), types.NamespacedName{Name: strings.TrimSpace(bk.Spec.EnvVars), Namespace: bk.Namespace}, configMap)
+			if err == nil {
+				str, ok := configMap.Data["PRAVEGA_CLUSTER_NAME"]
+				if ok {
+					pravegaClusterName := "PRAVEGA_CLUSTER_NAME" + str
```

Comment:
Why do we need 2 finalizers? Can we not do with just one???
Objective of both finalizers is the same to delete metadata from zookeeper.

Can we use a single finalizer with name like cleanupZookeeper_<PravegaClusterName> where <PravegaClusterName> is picked from the configmap.

Reply:
done